e2e-tests/IIoTPlatform-E2E-Tests/deploy/IoTHubPublisherDeployment.cs

```diff
@@ -30,23 +26,23 @@ public class IoTHubPublisherDeployment {
         /// </summary>
         public async Task<bool> CreateOrUpdateLayeredDeploymentAsync() {
             var isSuccessful = false;
-            var configuration = await CreateOrUpdateConfigurationAsync(new Configuration("__default-opcpublisher-standalone") {
-                Content = new ConfigurationContent { ModulesContent = CreateLayeredDeployment() },
-                TargetCondition = TargetConditionStandalone +
+            var configuration = await CreateOrUpdateConfigurationAsync(new Configuration(kDeploymentName) {
+                Content = new ConfigurationContent { ModulesContent = CreateDeploymentModules() },
+                TargetCondition = TestConstants.TargetCondition +
                     " AND tags.os = 'Linux'",
                 Priority = 1
-            }, true, new CancellationToken());
+            }, true, kDeploymentName, new CancellationToken());
```

Comment:
Please see my comment above about `IIoTHubDeployment.CreateOrUpdateLayeredDeploymentAsync()` method. It should itself get a `CancellationToken` which it would thenpass down to `CreateOrUpdateConfigurationAsync()` method instead of creating a new `CancellationToken`.

Reply:
done